projects/rocprofiler-compute/docs/how-to/analyze/data-dump.rst

```diff
@@ -0,0 +1,104 @@
+.. meta::
```

Comment:
I'd apologize here: we should have the doc for "--save-dfs" early.
Since you adding "data-dump" section here, I'd say we might want to include description for both db and csv format. And even more, the option could be in more unified fashion: --output [txt/csv/db]